### ISSUE TITLE
The about:preferences/options page does not follow the motif of this theme.

### ISSUE THREAD

**Issue by johngraciliano** [johngraciliano/simplewhite#3]
The options page is filled with lively colors! It's cheery but I do not think it is appropriate for this theme.  I am redecorating the options page to look like the addons page and most of the blue will turn into gray. 
As soon as I clean the code, I will upload a version with the changes as a branch.  I have yet to produce icons with the new size.  I am temporarily using inverted (negative) resized images of current icons.  A sample of the new (retro) look is at the bottom.
**Note:**  I want to change the _Advanced_ options icon to a five point star.  I think that change is long overdue after the change to the wizard's hat with a five point star in the default/classic theme.  
_Please comment!_

![options](https://cloud.githubusercontent.com/assets/13938911/10121854/6316c5f6-64ce-11e5-81f2-16ce1a2f52f9.png)

**Comment by lychichem** [comment]
If you use five point star, do you mind some users thinking it related to bookmark setting?

**Comment by johngraciliano** [comment]
@lychichem: I see they use stars for bookmarks in the default theme now (I use the default theme only for reference and very rarely).  I think of a star as how a general can be distinguished from other officials.  Stars are used to signify high rank (e.g., they are use for rating).  I think using a star for stating "Advanced" makes better sense than saying "Bookmark" yet I can think of that other connection.  It can certainly create a little confusion initially for those accustomed to the default theme.  But I think is better to have a star than to reuse the symbol for options in general.  I sincerely hope people will see it is the star in the wizards hat!
I already made new hand-coded svg icons and I am about to merge all changes to the main branch.  (I later found the location for png icons with the needed size).  I chose not to get rid of all the blue.  The links will remain blue (at least for now) because I am a bit concerned how oblique characters with light underline works with characters from Asian languages (I have no idea).  That plus the facts: (1) the css for this is used in a few other pages and (2) blue is use for links elsewhere in the theme.

**Comment by johngraciliano** [comment]
The changes are now in pre-release form.

**Comment by johngraciliano** [comment]
As @lychichem voiced a concern about the possible confusion a star may give as the Advance symbol I pondered deeply for an alternative.  Since the wizard's hat is too complicated and may look silly if simplified, I considered a top hat (like a magician's hat) but that would also be silly.  I considered several ideas, one of which was a kite.  As reconsidering the wheel symbol used in the options and thinking what would the most advanced version of that would be,  I found the radioactive danger symbol as a suitable alternative (I was looking for a symbol for atomic energy).  So the current alternatives for the Advanced options are the star (current) and the radioactive danger (proposed). Please anybody give your thoughts.

![star](https://cloud.githubusercontent.com/assets/13938911/10250120/fa4c2994-68f6-11e5-9691-b2b1b5ba46f4.png)

![radio](https://cloud.githubusercontent.com/assets/13938911/10250126/015b5e4e-68f7-11e5-9e18-d338945d32fe.png)

**Comment by lychichem** [comment]
New icon looks OK. Let's use it.

**Comment by johngraciliano** [comment]
**_Two Problems Pending:**_
1. **_IMPORTANT!**_ I am afraid my changes to the master branch did not consider the Darwin OS and the options page form the release candidate for 2.2b18.x1.03 will look bad in Apple computers.  That is because one of the two css files changed, _common.css_, has an separate version for the Darwin OS but not the other one, which would mean the icons may change while the style remain the same, and that is a problem because the new (retro) icons are black (and have a different size).  I have no means for testing, so help for testing on those systems is needed.   However, I will try to make the relevant changes to the code for another candidate.
2. I understand the _radioactive_ icon is OK.  I yet need to make\* a slightly smaller version for the old preferences (pop-up) window.  It has been neglected may be missing some settings, but it still works and I prefer it!

---

*Writing this code reminds me need to review trigonometry, yet I did not need to use trigonometric functions to code the _radioactive_ icon as I did with the _star_ because 60° angles are easy :)
